src/serialization/serialization.h

```diff
@@ -139,6 +149,25 @@ inline bool do_serialize(Archive &ar, bool &v)
   template <bool W, template <bool> class Archive>			\
   bool do_serialize_object(Archive<W> &ar){
 
+/*! \macro BEGIN_SERIALIZE_OBJECT_FN
+ *
+ * \brief Begins the environment of the DSL as a free function in object-style
+ * \detailed for describing how to
+ * serialize an of an archive type
+ */
+#define BEGIN_SERIALIZE_OBJECT_FN(stype)               \
```

Comment:
I agree that the comment needs some fixing - the goal is to allow a standalone function in a cpp or something? Difficult to say without any example uses.

Reply:
There's some sample uses in an earlier comment in this thread. Should I put it into the code?